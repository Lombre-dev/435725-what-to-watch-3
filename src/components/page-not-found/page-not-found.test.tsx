import * as React from 'react';
import {BrowserRouter} from 'react-router-dom';
import * as renderer from 'react-test-renderer';
import PageNotFound from './page-not-found';

describe(`<PageNotFound />`, () => {

  it(`render should be match markup`, () => {

    const result = renderer
      .create(<BrowserRouter>
        <PageNotFound />
      </BrowserRouter>)
      .toJSON();

    expect(result).toMatchSnapshot();
  });
});
