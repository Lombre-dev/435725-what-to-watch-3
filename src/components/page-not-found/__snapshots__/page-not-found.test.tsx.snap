// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> render should be match markup 1`] = `
<div
  className="page-content"
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "height": "100vh",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "color": "#dfcf77",
        "display": "flex",
        "fontSize": "30px",
        "fontWeight": "bold",
        "height": "100%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <p>
      Page not found
    </p>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
